sycl/doc/extensions/proposed/sycl_ext_oneapi_raw_kernel.asciidoc

```diff
@@ -0,0 +1,513 @@
+= sycl_ext_oneapi_raw_kernel
```

Comment:
I'm not sure about the name "raw" here.  I think referring to these as "kernel functions" would be easier for people to understand, because they're kernels defined as functions (as opposed to existing kernel function objects, or kernel objects).

We also talked about using "raw kernel argument" to mean something like "a kernel argument specified in terms of raw bytes", and the distinction between "an argument to a raw kernel" and "a raw argument to a kernel" is pretty subtle.

Reply:
Do you think it would be too hard to change the definition in the SYCL specification?  Arguably "kernel function" should probably have been "kernel function object" all along, because there's no way (without this extension) to turn a function into a kernel -- only function objects.

Reply:
How about "free function kernel" or "function-pointer kernel"? I agree that raw is bad. I'm OK with it as temporary solution.

Reply:
I'm OK with naming the extension "sycl_ext_oneapi_free_function_kernel".  I can also change occurrences of "raw kernel" in the text with "free function kernel".

Does that work for you @Pennycook?

Reply:
"Free function kernel" works for me.

Reply:
Extension renamed in 0f7f928433209be5a01399229714978ca0d957ef

Reply:
@gmlueck, please, update PR title/description too.

Reply:
> please, update PR title/description too.

Done.  Thanks for the reminder.